### DISCUSSION TITLE
Export to diagram to other tools like draw.io

### DISCUSSION THREAD

**Discussion by Jesse-efe** [xyflow/xyflow#2365] [New Features]
Hi React-flow Team.
Thanks for the work you are doing.

I like to suggest adding functionality to export created diagram to formats supported by other tools like draw.io (XML) and Visio (.vsdx) so that the diagram can be opened and edited in those tools.

This is important because if one is building an application that would require subscription using React-flow. One thing potential customers would consider before subscribing to the platform would be if they can easily move their diagrams to other platforms in case they no longer want to use the application.

If this is not something that you would consider adding in the nearest future, do you have suggestions on how this can be achieved?

I would like to know your thoughts on this topic.

Thanks.

**Comment by moklick** [comment]
Hey @Jesse-efe Thanks for your input. React Flow is a generic library. We don't know how your nodes are going to look like. If you want to export a flow to another platform you would need to write your own converter for that.

**Reply by Jesse-efe** [reply]
Ok thanks.
